web/config.py

```diff
@@ -761,6 +761,10 @@
     }
 ]
 
+# The claim which is used for the username. If the value is empty the
```

Comment:
This setting should be a part of individual oauth2 config dictionary, which will allow segregation of oauth2 claim setting in case of multiple oAuth2 provider.
Also please fix pep8 issue.

Reply:
Yes, I didn't thought about that. I've changed it and its now working for multiple OAuth2 provider.
Also the workflows should be green now.